lib/enhance-assert.js

```diff
@@ -12,23 +13,40 @@ module.exports.PATTERNS = [
 	't.regexTest(regex, contents, [message])'
 ];
 
-function enhanceAssert(assert) {
-	var empower = require('empower');
-	var powerAssertFormatter = require('power-assert-formatter');
-	var powerAssertRenderers = require('power-assert-renderers');
+module.exports.NON_ENHANCED_PATTERNS = [
+	't.pass([message])',
+	't.fail([message])',
+	't.throws(fn, [message])',
+	't.doesNotThrow(fn, [message])',
+	't.ifError(error, [message])'
+];
 
-	empower(assert,
-		powerAssertFormatter({
-			renderers: [
-				powerAssertRenderers.AssertionRenderer,
-				powerAssertRenderers.SuccinctRenderer
-			]
-		}),
+function enhanceAssert(opts) {
+	var empower = require('@jamestalmage/empower-core');
+	var enhanced = empower(
+		opts.assert,
 		{
-			destructive: true,
-			modifyMessageOnRethrow: true,
-			saveContextOnRethrow: false,
-			patterns: module.exports.PATTERNS
+			destructive: false,
+			onError: opts.onError,
+			onSuccess: opts.onSuccess,
+			patterns: module.exports.PATTERNS,
+			wrapOnlyPatterns: module.exports.NON_ENHANCED_PATTERNS
 		}
 	);
+
+	enhanced.AssertionError = opts.assert.AssertionError;
+
+	return enhanced;
+}
+
+function formatter() {
+	var powerAssertFormatter = require('power-assert-formatter');
+	var powerAssertRenderers = require('power-assert-renderers');
```

Comment:
Just put the `require` statements directly inside the array. They're inline anyways.